### ISSUE TITLE
Madrid 20200427

### ISSUE THREAD

**Issue by philipus** [victorvicpal/COVID19_es#17]
in 20200427 the numbers seems pretty large?!

**Comment by victorvicpal** [comment]
If you check the original data `data/pdf_data/Actualizacion_88_COVID-19.pdf` you can see that the values correspond to the values provided by the Minister.

**Comment by philipus** [comment]
yes, thank you for your hint. In documents the days before there is a comment on numbers from madrid (see below).

In the latest the comment is missing. Do you think one should/could distribute the numbers from 27th of april with the share of the other dates? I you wonna use the numbers for modelling

"Los datos de estas comunidades autónomas son datos de prevalencia (personas ingresadas a fecha de hoy). No reflejan el total de personas que han sido hospitalizadas o ingresadas en UCI a lo
largo del periodo de notificación por lo que no se puede realizar el sumatorio de todas las personas que han requerido hospitalización o ingreso en UCI en España"

**Comment by victorvicpal** [comment]
You're right.
I'll keep this issue open to fix the problem.
Thanks!
